lib/y-richtext.coffee

```diff
@@ -253,12 +310,15 @@ class YRichText extends BaseClass
       throw new Error "This part of code must not be reached!"
 
   insertHelper = (thisObj, position, content_array) ->
-    thisObj._model.getContent("characters").insertContents position, content_array
+    thisObj._model.getContent("characters").insertContents(
+      position, content_array)
 
   deleteHelper = (thisObj, position, length = 1) ->
     thisObj._model.getContent("characters").delete position, length
 
 if window?
+  window.Picker = Picker
+  window.ColorPicker = ColorPicker
```

Comment:
Please, don't pollute the window object! Every Application could have a "Picker" or a "ColorPicker". Then _they_ run into troubles and have to debug it. If you have to, you could put it on Y.RichText (e.g. Y.RichText.ColorPicker). But I currently see no reason to keep the ColorPicker in this file